src/validations/access/on-chain/client.ts

```diff
@@ -157,6 +124,24 @@ export const createOnChainClient = (
       return permissionOk()
     }
 
+    // Urns that need to be split into urn and tokenId
+    const extendedUrns = urnsToCheck
+      .filter((urn) => urn.type == 'blockchain-collection-v1-item' || urn.type == 'blockchain-collection-v2-item')
+      .map((urn) => urn.urn)
+
+    const extendedUrnsSplit = extendedUrns.map((extendedUrn) => {
+      const { assetUrn, tokenId } = getTokenIdAndAssetUrn(extendedUrn)
+      return { urn: assetUrn, tokenId }
+    })
+
+    // Urns that don't need to be split
+    const notExtendedUrns = urnsToCheck
+      .filter((urn) => urn.type != 'blockchain-collection-v1-item' && urn.type != 'blockchain-collection-v2-item')
```

Comment:
I think there is something wrong with the linter `!=` should be marked as an error